src/Command/ResetProductIndexCommand.php

```diff
@@ -111,4 +111,4 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $output->writeln(sprintf('<info>Command is already running</info>'));
         }
     }
-}
+}
```

Comment:
Can you revert this change?

Reply:
Yes

Reply:
Isn't this quite complicated: introducing an extra variable to check wether there are still productDocument waiting? In the ResetProductIndexCommand this problem is solved by adding `$this->elasticsearchManager->commit();` after the foreach. I guess it could be possible that in some cases commit will be called twice.